test/unit/node_stream_spec.js

```diff
@@ -30,7 +30,7 @@ describe('node_stream', function() {
   let server = null;
   let port = null;
   let pdf = url.parse(encodeURI('file://' + path.join(process.cwd(),
-                      './test/pdfs/tracemonkey.pdf'))).href;
+                      './test/pdfs/tracemonkeyWRONG_PATH.pdf'))).href;
```

Comment:
You can revert this change. Looking at the original issue you're solving, you can just remove https://github.com/mozilla/pdf.js/blob/master/test/unit/api_spec.js#L167-L171 and the unit tests should still pass if your change works.